tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -185,12 +185,12 @@ def name(self):
             return self.root_element.text
 
         def tap_icon(self):
-            expected_name = self.name
+            #expected_name = self.name
 
             # TODO bug 1043293 introduced a timing/tap race issue here
             time.sleep(0.5)
             self.root_element.tap(y=1)
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name.lower() == expected_name.lower())
+            self.wait_to_be_displayed()
```

Comment:
You can't do this, just leave this as is for now. This needs a more complicated fix.